matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/IMXEncrypting.kt

```diff
@@ -62,4 +62,11 @@ internal interface IMXEncrypting {
                            userId: String,
                            deviceId: String,
                            senderKey: String): Boolean
+
+    /**
+     * Ensure the outbound session
+     *
+     * @param usersInRoom the users in the room
+     */
+    suspend fun ensureOutboundSession(usersInRoom: List<String>)
```

Comment:
I'd recommand not putting this in this interface as it's specific to group encryption (not applicable to OLM)
Probably better to create a IMXGroupEncryption interface (like IMXWithHeldExtension) - we could even move other apis like reshareKey to this interface

Reply:
Done.